system/Database/ConnectionInterface.php

```diff
@@ -98,7 +100,7 @@ public function getVersion(): string;
      * Should automatically handle different connections for read/write
      * queries if needed.
      *
-     * @param mixed ...$binds
+     * @param array|string|null ...$binds
```

Comment:
This seems a bug.

```suggestion
     * @param array|string|null $binds
```

Reply:
Is there anything need to be fixed besides removing `...`?

Reply:
No, for now.

The difference of the parameter count may be a bug.